### DISCUSSION TITLE
Support for env file customization

### DISCUSSION THREAD

**Discussion by ZackarySantana** [remix-run/remix#3006] [Proposals]
When using "remix dev", it has built in dotenv support. The con to this, is that it must be named ".env". Would it be worth adding it to remix's config file as an optional value, and then adding the support for it in https://github.com/remix-run/remix/blob/0cb768fc1422a3f093e1fe8011694183920f381b/packages/remix-dev/env.ts ?

**Comment by kiliman** [comment]
The whole premise of Remix App Server is a batteries-included opinionated simple web server for your Remix app. If you need to customize it, the best option to use the Express adapter (since RAS is just Express).
